community/quassel/APKBUILD

```diff
@@ -1,8 +1,8 @@
 # Maintainer: Yves Schumann <yves@eisfair.org>
 
 pkgname=quassel
-pkgver=0.12.4
-pkgrel=3
+pkgver=0.12.5
+pkgrel=1
```

Comment:
pkgrel should be set to 0 when version is bumped.

Reply:
Please remove all the `|| return 1` from the code. It is legacy stuff.

Reply:
Please update the license so it is in line with the SPDX format:
https://spdx.org/licenses/
https://github.com/quassel/quassel/blob/master/COPYING

Reply:
Squash your commits and rename it and the PR to " community/quassel: upgrade to 0.12.5"

Reply:
Thx for the review and your comments. Everything is clear except the notes regarding the license. I have no idea what you mean with this or what I should change. Could you explain please?

Reply:
Alpine Linux project has decided to follow the SPDX standard naming convention. The current license mentioned in this APKBUILD file does not follow the convention it just says GPL. SPDX requires the specific variant, so if I need the COPYING file correct it should be `license="GPL-2.0-only GPL-3.0-only"`